### PR TITLE
Add missing not_found translations.

### PR DESCRIPTION
**Pull request by greut**
Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,6 +24,7 @@ module.exports = {
             create: 'Créer %{name}',
             delete: 'Supprimer %{name} #%{id}',
             dashboard: 'Tableau de bord',
+            not_found: 'Ooops, page manquante.'
```
Comment (djhi) [inline index.js]: Can you remove the `Ooops` ? I suggest: `Page manquante`

**Review by djhi — CHANGES_REQUESTED** [review]
Thanks :)

File: index.js (hunk 1 above)
Comment (greut) [reply index.js]: Sure :-)

**Comment by greut** [comment]
@djhi ping.

**Review by djhi — APPROVED** [review]

**Comment by fzaninotto** [comment]
published in version 1.9.0

**Pull request merged**